### PR TITLE
profile dev updates

### PR DESCRIPTION
**Pull request by aaronlippold**

### CODE REVIEW

File: src/courses/profile-dev-test/11.md [hunk 1]
```diff
@@ -1,20 +1,72 @@
...
+## Types of Profile Updates
 
-3. **Release Readiness Is Predefined:** A Benchmark is considered 'ready for release' when it meets the expected thresholds, hardening, and validation results. Don't be overwhelmed by the multitude of changes across the files. Instead, focus on the specific requirement you are working on. Understand its expected failure and success states on each of the target testing platforms. This approach prevents you from being overwhelmed and provides solid pivot points as you work through the implementation of the automated tests for each requirement and its 'contexts'.
+Security benchmark profiles require regular updates to maintain their effectiveness. Let's explore the three main types of updates:
 
-4. **Use Vendor-Managed Standard Releases:** When setting up a test suite, prioritize using vendor-managed standard releases for software installations and baseline configurations. This should be the starting point for both 'vanilla' and 'hardening' workflows. This approach ensures that your initial and ongoing testing, hardening, and validation closely mirror the real-world usage scenarios of your end-users.
+<table>
+  <thead>
```
Comment (aaronlippold) [inline src/courses/profile-dev-test/11.md:21]: Why isn't this a markdown table?

Comment (Amndeep7) [reply src/courses/profile-dev-test/11.md:21]: cause markdown tables can't do headers on the left hand side properly

**Review by aaronlippold — COMMENTED** [review]
there are a few little things but I think this is good to go

File: src/courses/profile-dev-test/17.md [hunk 2]
```diff
@@ -1,33 +1,90 @@
...
+- The purpose and function of Test Kitchen's converge stage
+- How to use different infrastructure configurations
+- The differences between vanilla and hardened environments
+
 ## Test Kitchen Converge Stage
 
-The `converge` stage uses Ansible Playbooks from the Ansible Lockdown project to apply hardening configurations, specifically the RHEL8-STIG playbook, and RedHat managed containers.
+The `converge` stage applies system configurations using infrastructure as code (IaC) tools. This crucial stage transforms your environment from a base state to your desired configuration.
+
+::: info Let 'em cook
```
Comment (wdower) [inline src/courses/profile-dev-test/17.md:20]: 🤔

**Review by wdower — APPROVED** [review]

**Pull request merged**